addons/web/static/src/js/fields/field_registry.js

```diff
@@ -21,7 +21,7 @@ registry
     .add('abstract', AbstractField)
     .add('input', basic_fields.InputField)
     .add('integer', basic_fields.FieldInteger)
-    .add('boolean', basic_fields.FieldBoolean)
+    //.add('boolean', basic_fields.FieldBoolean)
```

Comment:
either remove the line, or add a note saying why you commented it (i suggest the second one).